lms/static/sass/course/ccx_coach/_dashboard.scss

```diff
@@ -44,12 +49,99 @@ table.ccx-schedule {
     border: 1px solid #cbcbcb;
     padding: 15px;
     margin-bottom: 20px;
+    button {
+        display: inline-block;
+        vertical-align: middle;
+        margin-right: .3125rem;
+        margin-bottom: .625rem;
```

Comment:
Why are you using `rem`? this is not a convention we use, we use pixels in these files, or better yet, as a percentage of `$baseline`. Look throughout other .scss files to see how we're using css.